src/applications/disability-benefits/526EZ/helpers.jsx

```diff
@@ -385,7 +385,7 @@ export const veteranInformationViewField = (data) => {
         <p>Date of birth: <DateWidget value={dateOfBirth} options={{ monthYear: false }}/></p>
         <p>Gender: {genderLabels[gender]}</p>
       </div>
-      <p><strong>Note:</strong> If something doesn’t look right and you need to update your details, please call Veterans Benefits Assistance at <a href="tel:1-800-827-1000">1-800-827-1000</a>, Monday through Friday, 8:00 a.m. to 9:00 p.m. (ET).</p>
+      <p><strong>Note:</strong> If something doesn’t look right and you need to update your details, please call Veterans Benefits Assistance at <a href="tel:1-800-827-1000">1-800-827-1000</a>, Monday – Friday, 8:00 a.m. to 9:00 p.m. (ET).</p>
```

Comment:
It looks like we're not very consistent about how we write this actually, several different combination of spaces and dash types appear to be in use, is there a preference among these @peggygannon? If so, we could set up a ticket to make these consistent:
`Monday &#8211; Friday`
`Monday – Friday`
`Monday — Friday`

Reply:
According to http://www.getitwriteonline.com/archive/091502enem.htm, it should be an `en dash` (the middle one).